### PR TITLE
Adds Beefmen

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Approval
image

This PR adds Fulpstation's Beefman species, the wiki page (cannot confirm if it's outdated or not) is here: https://wiki.fulp.gg/en/beefmen

The HackMD for the species can also be found here: https://hackmd.io/mpEyXEt9QhaqlthsPDTy6w

Pros of the species:

- is faster and TALL
- has bluespace prophet-like trauma that allows you to TELEPORT AROUND (unless seen (this includes tracking implants) (this excludes silicon))
- is RED and BEEFY
- can TEAR OFF LIMBS AND TONGUES and REPLACE THEM at will using SLABS OF MEAT

Cons of being MEATY:

- takes more damage overall
- bleeds out when warm (their base temperature is 20C, lower than default, so can heat up easily) - This can be countered by burn damage. Hit yourself with a match, cig, lighter, laser gun, welder, to keep yourself from bleeding out!
- bleeds out from consuming salt (optimal revenant target)
- Has BRAIN DAMAGE
- Lacks DNA
- Cannot disguise easily, you're a large red slab of meat

image
image

## Why It's Good For The Game

It's a new (to TG) species that's been running for a few years now on fulp so I think it's had enough time to tell if it's a good species or not.

It's a species with a lot of unique mechanics and features that aren't generally too overpowered/underpowered but gives certain advantages/disadvantages, all without being an animal species.

Turns limb/blood management into a more chaotic mess & adds more to dangerous environments (areas that are too hot will make beefmen bleed, beefmen can raid the kitchen for new limbs, ect).

## Changelog

:cl: TheSwain, Guillaume Prata, JohnFulpWillard, Sprites by Horatio22 and tommysalami3
add: Beefmen.
/:cl:

### CODE REVIEW

File: code/datums/brain_damage/bluespace_prophet.dm [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+	else
+		user.visible_message(span_warning("[user] [slip_out_message], ending up exactly where they left."), span_notice("...and find yourself where you started?"))
+
+/obj/effect/client_image_holder/bluespace_stream/attack_tk(mob/user)
+	to_chat(user, span_warning("\The [src] actively rejects your mind, and the bluespace energies surrounding it disrupt your telekinesis!"))
+	return COMPONENT_CANCEL_ATTACK_CHAIN
+
+///Phobetor trauma, given to Beefmen
+/datum/brain_trauma/special/bluespace_prophet/phobetor
+	name = "Sleepless Dreamer"
```
Comment (Fikou) [inline code/datums/brain_damage/bluespace_prophet.dm:120]: lifeweb

Comment (JohnFulpWillard) [reply code/datums/brain_damage/bluespace_prophet.dm:120]: what did they mean by this

**Comment by notkripton** [comment]
Oh god

**Comment by optimumtact** [comment]
No, and addtionally, nova's right to approve any Pr's is revoked

**Comment by OneAsianTortoise** [comment]
RIP Bozo

**Comment by JohnFulpWillard** [comment]
☹️

**Comment by fishingsman** [comment]
i dont approve this pr

**Comment by Xander3359** [comment]
Why was this speed closed?

**Comment by Mothblocks** [comment]
theres no such thing as speed closing. the 24h rule only applies to merging. if you know something will never be merged there is no reason to make the op think otherwise.

**Pull request closed without merging**